resources/views/components/meta-panel/select-filter.blade.php

```diff
@@ -10,9 +10,7 @@
 ?>
 
 <script>
-function handleSelectFilterChanged(event) {
-    const kind = '{{ $kind }}';
-
+function handleSelectFilterChanged(kind, event) {
```

Comment:
nit: Conventionally in JS ecosystem handler functions, `event` is always the first arg.

Reply:
done